### ISSUE TITLE
Not able to access the pixel streaming in office laptop (WebRTC connection negotiated)

### ISSUE THREAD

**Issue by DenisTensorWorks** [EpicGames/PixelStreamingInfrastructure#14]
Originally opened by [@rahulzephyr](https://github.com/rahulzephyr)

**UE Version:**
E.g. UE 5.2

**Frontend Version:**
E.g. UE5.2

**Problem component**
E.g. Signalling Server, Pixel Streaming

**Description**
I have hosted the pixel streaming in the AWS EC2 instance (Windows server 2022, g4dn.2x large family instance type). Using the public IP address of EC2 I could access the streaming in my private devices (personal laptop and mobile device) but not able to access the same in my office laptop. As this is part of my office project it is expected to access the streaming in the office laptop as well. 

In my laptop I have created the inbound and outbound rule allowing all ports as well but still no luck :( 

I cannot access it in my office network or if my laptop is connected to a VPN. I don't know if I make it as an HTTPS-secured connection, it will work.

**Error:**
I am getting the error like "WEBRTC CONNECTION NEGOTIATED"

![image](https://github.com/EpicGames/PixelStreamingInfrastructure/assets/53375028/9de88c98-69e9-4b53-9cfa-d98bf3decac2)

Since I was getting error in port 80 while running the file Start_WithTURN_SignallingServer, I changed it to HttpPort: 90 in config.json (earlier even when it was working in port 80 the issue was there).

Is this something I need to check with my office IT security team to unblock something or some work around can be done in Pixel streaming itself?

It would be really helpful if someone help me resolve this.
Thanks in advance!!

---

Moved from https://github.com/EpicGames/PixelStreamingInfrastructure/issues/460

**Comment by lukehb** [comment]
@rahulzephyr Are you using a TURN server? These are sometimes required when you are streaming to devices in enterprise, corporate, or restricted networks such as offices and schools. It is not uncommon for your administrator to put locked down networking on a work laptop etc.

**Comment by luming222** [comment]
@lukehb 
I'm also having the same problem, it's on a front-end computer that can't connect to a public network, and I want to ask if it's a network problem that doesn't work on the front-end, we deploy UE5 Pixel Streaming on another machine in the LAN If this issue can be fixed, I'm very grateful

**Comment by lukehb** [comment]
@luming222 Your issue sounds a little different than the one described here. Would you mind opening a new ticket with steps explaining how to reproduce it?

**Comment by luming222** [comment]
@lukehb I have put the relevant questions on # 157. If it's convenient for you to take a look, thank you very much

**Comment by bootsch** [comment]
Running into the same issue (WebRTC connection negotiated). Running the SignallingWebserver with turn or stun does not solve the issue for us. All works fine streaming from a local machine and accessing with a brower on another local machine. Any other networks shows this message. Any suggestions to fix this?
Using UE 5.5 and main branch of this repository

**Comment by bootsch** [comment]
After https setup and running with stun it works for us.
